app/src/main/res/values/strings.xml

```diff
@@ -8,12 +8,13 @@
     <string name="create_stage">create stage</string>
     <string name="twitter_connect">Sign in with Twitter</string>
     <string name="sync_stage_user">sync stages</string>
-
-    <string name="sound_description"></string>
+    <string name="stage_count">%1$d/%2$d</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.stage_count` appears to be unused
 Unused resources make applications larger and slow down builds.